modules/nw-ingress-controller-configuration-proxy-protocol.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+[id="nw-ingress-controller-configuration-proxy-protocol_{context}"]
+= Configuring PROXY protocol for an Ingress Controller
+As a cluster administrator, you can use PROXY protocol to enhance communication between the Ingress Controller and an external load balancer. When you publish an Ingress Controller with a load balancer, the connections that the Ingress Controller receives have the load balancer's source address, instead of the original client address. The PROXY protocol enables a load balancer to preserve the original client addresses and include them when forwarding connections to an Ingress Controller, which can then use the original client addresses for logging.
+
+You can configure an Ingress Controller to use PROXY protocol when the endpoint publishing stratey type is `HostNetwork` or `NodePortService`. Set the `spec.endpointPublishingStrategy.nodePort.protocol` or `spec.endpointPublishingStrategy.hostNetwork.protocol` field to `PROXY`.
+
+[IMPORTANT]
+====
+This feature is supported only in non-cloud deployments. This restriction is because when {product-title} runs in a cloud platform, and an IngressController specifies that a service load-balancer should be used, the ingress operator configures the load-balancer service and infers, based on the specific platform, whether or not the PROXY protocol is needed for preserving source addresses.
+====
+
+[WARNING]
+====
+Configure both the Ingress Controller and the load balancer to use PROXY protocol to prevent connection failure. Connections will fail if both configurations are not the same.
```

Comment:
Suggest reordering first sentence to get the goal first ("To prevent connection failure, ..."). 

Similarly, it would be good to get the conditional in the second sentence upfront ("If the configurations are inconsistent, ..."). Also avoid future tense here if you can :) But the more I think about this, the second sentence doesn't really add any information, so you might just want to leave it at the first sentence.